katdal/sensordata.py

```diff
@@ -111,22 +111,22 @@ def __repr__(self):
                (self.__class__.__name__, self.name, id(self))
 
 
-class SimpleSensorData(SensorData):
+class SimpleSensorGetter(SensorGetter):
     """Raw sensor data held in memory.
```

Comment:
While the abstract `SensorData` warns against caching sensor values, that's exactly what `SimpleSensorGetter` does...
And it should 😄

Reply:
I wouldn't call it "caching", since that implies a copy of data that can be retrieved from elsewhere. It holds the master copy in memory.

Reply:
Sure, I thought about that too, but the `SensorGetter` docstring also says "... but does not store values itself." :-)
I don't think there is anything to change here. I added the comment to the PR mostly for my future self...